src/Command/Pull/PullCommandBase.php

```diff
@@ -88,6 +89,7 @@ protected function pullDatabase(InputInterface $input, OutputInterface $output):
     $database = $this->determineSourceDatabase($acquia_cloud_client, $source_environment);
     $this->checklist->addItem('Importing Drupal database copy from the Cloud Platform');
     $this->importRemoteDatabase($source_environment, $database, $this->getOutputCallback($output, $this->checklist));
+    $this->createDbSettingsFile($source_environment, $database);
```

Comment:
You should verify that this is an IDE since it is possible to run this command elsewhere.

Reply:
Ah right, forgot about Lando...